controllers/datamovement_controller_test.go

```diff
@@ -254,7 +256,7 @@ var _ = Describe("Data Movement Test", func() {
 						cmd = dm.Status.CommandStatus.Command
 					}
 					return cmd
-				}).Should(Equal(dmCfgProfile.Command))
+				}).Should(Equal(cmdBashPrefix + dmCfgProfile.Command))
```

Comment:
I thought this too, but it seems like go is negating the need for that.

Here is a command that I'm setting in the config map:
```yaml
command: date && time mpirun --allow-run-as-root --hostfile $HOSTFILE hostname && date
```
You can see it's doing a bunch of bash-like things: 3 commands, time is wrapping mpirun which is wrapping hostname.

Running this as part of data movement:
```
2023-05-31T16:27:26.517Z	INFO	Running Command	{"controller": "nnfdatamovement", "controllerGroup": "nnf.cray.hpe.com", "controllerKind": "NnfDataMovement", "NnfDataMovement": {"name":"nnf-dm-pd9x5","namespace":"nnf-dm-system"}, "namespace": "nnf-dm-system", "name": "nnf-dm-pd9x5", "reconcileID": "cfc24100-ee50-4a12-a25e-e2ec6a14a4a5", "cmd": "/bin/bash -c date && time mpirun --allow-run-as-root --hostfile /tmp/nnf-dm-pd9x5/hostfile hostname && date"}
2023-05-31T16:27:29.940Z	INFO	Data movement operation completed	{"controller": "nnfdatamovement", "controllerGroup": "nnf.cray.hpe.com", "controllerKind": "NnfDataMovement", "NnfDataMovement": {"name":"nnf-dm-pd9x5","namespace":"nnf-dm-system"}, "namespace": "nnf-dm-system", "name": "nnf-dm-pd9x5", "reconcileID": "cfc24100-ee50-4a12-a25e-e2ec6a14a4a5", "cmdStatus": {"command":"/bin/bash -c date && time mpirun --allow-run-as-root --hostfile /tmp/nnf-dm-pd9x5/hostfile hostname && date","elapsedTime":"3.406380696s","lastMessage":"Wed May 31 16:27:29 UTC 2023\n","lastMessageTime":"2023-05-31T16:27:29.932613Z"}}
2023-05-31T16:27:29.941Z	INFO	Data movement operation output	{"controller": "nnfdatamovement", "controllerGroup": "nnf.cray.hpe.com", "controllerKind": "NnfDataMovement", "NnfDataMovement": {"name":"nnf-dm-pd9x5","namespace":"nnf-dm-system"}, "namespace": "nnf-dm-system", "name": "nnf-dm-pd9x5", "reconcileID": "cfc24100-ee50-4a12-a25e-e2ec6a14a4a5", "output": "Wed May 31 16:27:26 UTC 2023\nWarning: Permanently added '[10-42-2-89.dm.nnf-dm-system]:2222,[10.42.2.89]:2222' (ECDSA) to the list of known hosts.\r\nnnf-dm-worker-kzqxk\nnnf-dm-worker-kzqxk\nnnf-dm-worker-kzqxk\nnnf-dm-worker-kzqxk\nnnf-dm-worker-kzqxk\nnnf-dm-worker-kzqxk\nnnf-dm-worker-kzqxk\nnnf-dm-worker-kzqxk\n\nreal\t0m3.401s\nuser\t0m0.078s\nsys\t0m0.283s\nWed May 31 16:27:29 UTC 2023\n"}
```

The last line includes the output of the command. It's printing the date, then mpirun output, then time output, and the date again.

If I include single or double quotes in the command, it will fail.